packages/react-composites/tests/browser/call/hermetic/UnsupportedBrowserPage.test.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import type { EnvironmentInfo } from '@azure/communication-calling';
```

Comment:
Is this file not throwing errors in stable?
Due to unused imports

Reply:
it is not because one of the helper functions in the test is using the type so this should be fine.